x/incentive/types/keys.go

```diff
@@ -30,6 +30,12 @@ const (
 	// Eden Boost denom
 	EdenB = "uedenb"
 
+	// Lp Elys
+	LpElys = "ulpelys"
+
+	// Molecule Elys
+	MElys = "umelys"
```

Comment:
@kenta-elys please define those new denoms in the config.yml file as well under bank > metadata_denom

Reply:
Okay sure. it is not yet confirmed. Once it gets approved by team I'll definitely define them. Thanks!

Reply:
@kenta-elys I need those changes to approve though ;-)

Reply:
@kenta-elys I am not seeing the config.yml changes I requested, any progress on that?

Reply:
Those lp token definition got removed as we will have unique lp token per pool. Meaning not one for all liquidity pools. So it couldn't be predefined and we could get it through liquidity pool module. So those hard coded definitions got removed and no need to update config.yml.